src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -338,6 +338,14 @@
     short-summary: The ARM Id or name of the subnet for the vnet. If omitted 'default' will be used
   - name: --protocol-types
     short-summary: Space seperated list of protocols that the volume can use
+  - name: --volume-type
+    short-summary: Whether the volume should be a data protection volume ("DataProtection")
+  - name: --endpoint-type
+    short-summary: Whether the volume is source ("src") or destination ("dst")
+  - name: --remote-volume-resource-id
+    short-summary: The volume id of the remote volume of the replication (the destination for "src" volume endpoints and the source for "dst" endpoints)
+  - name: --replication-schedule
+    short-summary: The replication schedule, e.g. "_10minutely"
```

Comment:
I have some questions for the paramwter:
1. So the available value will be something like "_10minutely"? and how can I represent seconds? hours? 
2. Is it something like interval?

Reply:
Yes this in interval  hourly, daily, weekly, monthly.